App/Redux/CameraRollRedux.ts

```diff
@@ -30,13 +51,16 @@ export type QueriedPhotosMap = {
 export type CameraRollState = {
   readonly initialized: boolean
   readonly querying: boolean
-  readonly queriedPhotos: QueriedPhotosMap
+  readonly queriedPhotos: QueriedPhotosMap,
+  // A map of threads and their pending images from pendingImageShares
+  readonly pendingShares: { [index:string] : Array<TextileTypes.SharedImage> }
```

Comment:
pendingShares only hang around while the image is being processed prior to upload. once it is added to the upload queue, this guy gets removed